jsdrupal/drupal-admin-ui
# Render Default layout so that route change doesn't cause remounting

## Issue

https://github.com/jsdrupal/drupal-admin-ui/issues/62

## Screenshot / UI changes

No UI changes.

## Testing instructions

Ensure that menu links are requested between route changes. This can be tested on the content page so that you first load the content list, and then click the add content button. Make sure that there's only one request for menu links.
